source/scripting_v3/GTA/Entities/Peds/Ped.cs

```diff
@@ -215,6 +215,29 @@ public float WetnessHeight
 
 		#region Configuration
 
+		/// <summary>
+		/// Gets or sets a value indicating whether this <see cref="Entity"/> is persistent.
+		/// Unlike <see cref="Entity.IsPersistent"/>, setting a value does not affect assigned tasks.
+		/// </summary>
+		/// <value>
+		/// <see langword="true" /> if this <see cref="Entity"/> is persistent; otherwise, <see langword="false" />.
+		/// </value>
+		public bool IsPersistentNoClearTask
```

Comment:
This is a bit weird. Maybe just overwrite `IsPersistent` for peds and always do this (I'd be surprised if there was a script out there that relied on the task list being cleared by setting this).

Reply:
I would rather create `SetIsPersistentNoClearTask` method than overwrite `IsPersistent`, because overwrite `IsPersistent` can break compatibility with scripts that call `Ped.IsPersistent = false` instead of `Ped.MarkAsNoLongerNeeded()`.

Reply:
Btw, I wasn't suprised at all when I knew `SET_ENTITY_AS_NO_LONGER_NEEDED` clears peds’ tasks because I'd knew the equivalent opcode `01C2` for SCM scripts for 3D Era games also clears peds’ tasks.

Reply:
Fair enough.

Reply:
I would make Ped.IsPersistent not change assigned tasks in v4 API, but not in v3 in favor of compatibility.